modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-10_{context}"]
+= Release notes for {pipelines-title} General Availability 1.10
+
+With this update, {pipelines-title} General Availability (GA) 1.10 is available on {product-title} 4.11 and later versions.
+
+[id="new-features-1-10_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.10.
+
+[id="pipelines-new-features-1-10_{context}"]
+=== Pipelines
+
+* With this update, you can specify environment variables in a `PipelineRun` or `TaskRun` pod template to override or append the variables that are configured in a task or step. Also, you can specify environment variables in a default pod template to use those variables globally for all `PipelineRuns` and `TaskRuns`. This update also adds a new default configuration `forbidden-envs` to filter the environment variables while propagating from pod templates.
+* With this update, custom tasks in pipelines are enabled by default. 
++
+[NOTE]
+====
+To disable this update, set the `enable-custom-tasks` flag to `false` in the `feature-flags` config custom resource.
+====
+
+* This update supports the `v1beta1.CustomRun` API version for custom tasks.
+* This update adds support for the `PipelineRun` reconciler to create a custom run. For example, custom `TaskRuns` created from `PipelineRuns` can now use API version `v1beta1.CustomRun` instead of `v1alpha1.Run`, if the `custom-task-version` feature flag is set to `v1beta1`, instead of the default value `v1alpha1`. 
++
+[NOTE]
+====
+You need to update the custom task controller to listen for the `*v1beta1.CustomRun` API version instead of `*v1alpha1.Run` in order to respond to `v1beta1.CustomRun` requests.
+====
+
+* This update adds a new field `retries` into `v1beta1.TaskRun` and `v1.TaskRun` specifications.
+
+
+[id="triggers-new-features-1-10_{context}"]
+=== Triggers
+
+* With this update, triggers support the creation of v1 `Pipelines`, `Tasks`, `PipelineRuns`, `TaskRuns`, and v1beta1 `CustomRuns` objects.
+* With this update, GitHub Interceptor blocks a pull request trigger from being executed unless invoked by an owner or with a configurable comment by an owner. 
++
+[NOTE]
+====
+To enable or disable this update, set the value of the `githubOwners` parameter to `true` or `false` in the GitHub Interceptor configuration file.
+====
+
+* With this update, GitHub Interceptor has the ability to add a comma delimited list of all files that have changed for the push and pull request events. The list of changed files is added to the `changed_files` property of the event payload in the top-level extensions field.
+* This update changes TLS `MinVersion` to `tls.VersionTLS12` so that triggers run on {product-title} when the Federal Information Processing Standards (FIPS) mode is enabled.
```

Comment:
```suggestion
* This update changes the `MinVersion` of TLS to `tls.VersionTLS12` so that triggers run on {product-title} when the Federal Information Processing Standards (FIPS) mode is enabled.
```